sources/packages/backend/apps/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -37,6 +39,10 @@ export class CreateInstitutionAPIInDTO {
   website: string;
   @IsNotEmpty()
   regulatingBody: string;
+  @ValidateIf((e) => e.regulatingBody === "other")
+  @IsNotEmpty()
+  @MaxLength(OTHER_REGULATING_BODY_MAX_LENGTH)
```

Comment:
Since we created the below file I believe that we can keep adding these constants there, unless they are beyond the API scope.
```
sources\packages\backend\apps\api\src\constants\class-validator-constants.ts
```
@andrepestana-aot @ann-aot @dheepak-aot FYI